### PR TITLE
:bug: Fix weapons list in game

### PR DESCRIPTION
**Pull request by DrakenLine**

### CODE REVIEW

**Comment by DrakenLine** [comment]
Task linked: [CU-860rtxb98 Fix weapons list in game](https://app.clickup.com/t/860rtxb98)

File: lib/shared/Weapons/weapon_widget.dart [hunk 1]
```diff
@@ -99,35 +103,38 @@ class _WeaponWidgetState extends State<WeaponWidget> {
...
-                  child: ClipRRect(
-                    borderRadius: BorderRadius.circular(8),
-                    child: Image.network(widget.weapon.src),
+                  // foregroundDecoration: BoxDecoration(
+                  //   // color: setForegroundDecoration(gameState),
+                  //   backgroundBlendMode: (widget.isLocked || !widget.isSelected)
+                  //       ? BlendMode.saturation
+                  //       : null,
+                  //   borderRadius: BorderRadius.circular(8),
+                  // ),
```
Comment (MaximeWrlz) [inline lib/shared/Weapons/weapon_widget.dart:112]: remove dead code

File: lib/pages/home/home.page.dart [hunk 2]
```diff
@@ -114,21 +114,21 @@ class HomePage extends ConsumerWidget {
...
+                    //     function: () {
+                    //       authenticateOrNavigateToProfile(context);
+                    //     },
+                    //     icon: IconWidget(
+                    //         name: IconName.gearWheel,
+                    //         size: IconSize.large,
+                    //         fillColor: BSHexaColors().white,
+                    //         borderColor: BSHexaColors().black),
+                    //   ),
+                    // ),
```
Comment (MaximeWrlz) [inline lib/pages/home/home.page.dart:131]: we can remove this for the moment

File: lib/services/socket.service.dart [hunk 3]
```diff
@@ -155,6 +155,8 @@ class WebSocketService {
           '[SOCKETS] ' 'Next turn : ${SocketsEventsListened.gameOver.value}',
         );
       });
+
+
```
Comment (MaximeWrlz) [inline lib/services/socket.service.dart:159]: remove one line break

**Review by MaximeWrlz — APPROVED** [review]

**Review by AlexandreMichelix — APPROVED** [review]

**Pull request merged**